lib/omni_exchange/providers/open_exchange_rates.rb

```diff
@@ -4,33 +4,67 @@
 
 module OmniExchange
   class OpenExchangeRates < Provider
-    ENDPOINT_URL = 'https://openexchangerates.org/api/latest.json'
-
-    # This method returns the exchange rate, the rate at which the smallest unit of one currency (the base currency)
-    #   will be exchanged for another currency (the target currency), from Open Exchange Rate's API.
-    #   This method is called in the OmniExchange.exchange_currency method.
-    #
-    # @param base_currency: [String] the ISO Currency Code of the currency that you're exchanging from. ie. "USD", "JPY"
-    # @param target_currency: [String] the ISO Currency Code of the currency that you're exchanging to. ie. "EUR", "KRW"
-    # @ return [BigDecimal] an exchange rate is returned as a BigDecimal for precise calculation since this exchange
-    #   rate will be used to calculate an convert an exchange of currencies. However, an exception will be raised
-    #   if there is a timeout while connecting to xe.com or a timeout while reading Open Exchange Rate's API.
-    def self.get_exchange_rate(base_currency:, target_currency:)
-      config = OmniExchange.configuration.provider_config[:open_exchange_rates]
-      app_id = config[:app_id]
-
-      api = Faraday.new(OmniExchange::OpenExchangeRates::ENDPOINT_URL)
-
-      response = api.get do |req|
-        req.url "?app_id=#{app_id}&base=#{base_currency}"
-        req.options.timeout = config[:read_timeout] || OmniExchange::Configuration::DEFAULT_READ_TIMEOUT
-        req.options.open_timeout = config[:connect_timeout] || OmniExchange::Configuration::DEFAULT_CONNECTION_TIMEOUT
+    ENDPOINT_URL = 'https://openexchangerates.org/api/'
+
+    class << self
+      # This method returns the exchange rate, the rate at which the smallest unit of one currency (the base currency)
+      #   will be exchanged for another currency (the target currency), from Open Exchange Rate's API.
+      #   This method is called in the OmniExchange.exchange_currency method.
+      #
+      # @param base_currency: [String] the ISO Currency Code of the currency that you're exchanging from. ie. "USD", "JPY"
+      # @param target_currency: [String] the ISO Currency Code of the currency that you're exchanging to. ie. "EUR", "KRW"
+      # @ return [BigDecimal] an exchange rate is returned as a BigDecimal for precise calculation since this exchange
+      #   rate will be used to calculate an convert an exchange of currencies. However, an exception will be raised
+      #   if there is a timeout while connecting to xe.com or a timeout while reading Open Exchange Rate's API.
+      def get_exchange_rate(base_currency:, target_currency:)
+        body = api_get do |req|
+          req.url 'latest.json'
+          req.params['base'] = base_currency
+          req.params['symbols'] = target_currency
```

Comment:
We weren't passing the symbols parameter before but I added it here to save them some work. If we don't add it, open exchange rates will return a huge list with exchange rates for all currencies. We don't need them to do so much work when we only need one of them.

This meant I had to update the cassettes too as the request now has an extra parameter in it.